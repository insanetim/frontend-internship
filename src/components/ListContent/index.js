import ListContent from './component'

export default ListContent
