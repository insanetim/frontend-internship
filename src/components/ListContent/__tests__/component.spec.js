import { shallow } from 'enzyme'

import ListContent from '../component'

it('matches snapshot', () => {
  const mockedMovies = [
    {
      id: 1,
      title: 'test/title',
      overview: 'test/overview',
      poster_path: 'test/poster_path'
    }
  ]
  const component = shallow(<ListContent movies={mockedMovies} />)

  expect(component).toMatchSnapshot()
})

it('matches snapshot without movies', () => {
  const component = shallow(<ListContent />)

  expect(component).toMatchSnapshot()
})
