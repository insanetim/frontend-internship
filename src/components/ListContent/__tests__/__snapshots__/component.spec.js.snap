// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Row
  gutter={16}
  justify="center"
>
  <Col
    span={20}
  >
    <Row
      gutter={
        {
          "lg": 32,
          "md": 24,
          "sm": 16,
          "xs": 8,
        }
      }
    >
      <Col
        key="1"
        lg={
          {
            "span": 8,
          }
        }
        md={
          {
            "span": 8,
          }
        }
        sm={
          {
            "span": 12,
          }
        }
        xl={
          {
            "span": 6,
          }
        }
        xs={
          {
            "span": 24,
          }
        }
      >
        <Movie
          actions={[]}
          movie={
            {
              "id": 1,
              "overview": "test/overview",
              "poster_path": "test/poster_path",
              "title": "test/title",
            }
          }
        />
      </Col>
    </Row>
  </Col>
</Row>
`;

exports[`matches snapshot without movies 1`] = `
<Empty
  description="No results"
/>
`;
